File: 5. FrenchTeacherApp/app/src/main/java/com/example/frenchteacherapp/MainActivity.java
package com.example.frenchteacherapp;

import androidx.appcompat.app.AppCompatActivity;

import android.media.MediaPlayer;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity implements View.OnClickListener {

    Button blackBtn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        blackBtn = findViewById(R.id.blackBtn);
        blackBtn.setOnClickListener(this);
    }

    @Override
    public void onClick(View view) {

        int buttonids = view.getId();

        if (buttonids == R.id.blackBtn){
            playSound(R.raw.black);
        }

    }


    public void playSound(int id){
        MediaPlayer mediaPlayer = MediaPlayer.create(getApplicationContext(),id);
        mediaPlayer.start();
    }
}

File: __Kotlin Apps/French Teacher App/app/src/main/java/com/example/frenchteacherapp/MainActivity.kt
package com.example.frenchteacherapp

import android.media.MediaPlayer
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.view.View
import android.widget.Button

class MainActivity : AppCompatActivity() {

    private  val TAG = "MainActivity"
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)



    }

    fun sayColorName(view: View){
        val clickedButton : Button = view as Button

        val resourceId = resources.getIdentifier(clickedButton.tag.toString(), "raw", packageName)
        Log.i(TAG,clickedButton.tag.toString())
        val mediaPlayer = MediaPlayer.create(this, resourceId)

        mediaPlayer.start()

    }


}

